packages/starlight/style/props.css

```
@@ -93,7 +93,7 @@
 
   --sl-text-body: var(--sl-text-base);
   --sl-text-body-sm: var(--sl-text-xs);
-  --sl-text-code: var(--sl-text-base);
+  --sl-text-code: var(--sl-text-xs);
```

Thanks @lloydjatkinson!

I think 13px here might be just too small. Let’s try `sm` which is 14px:

```suggestion
  --sl-text-code: var(--sl-text-sm);
```